src/Store/CommandStoreConnector.re

```diff
@@ -106,6 +106,12 @@ let createDefaultCommands = getState => {
         ~action=CopyActiveFilepathToClipboard,
         (),
       ),
+      Command.create(
+        ~category=Some("System"),
+        ~name="Add Oni2 to System PATH", /* TODO: This should be dynamic (Add/Remove) */
+        ~action=Command("system.addToPath"),
```

Comment:
We can gate the command here with `~enabled=Revery.Environment.os == Revery.Environment.mac`.